examples/pxScene2d/src/pxContext.h

```diff
@@ -127,17 +166,28 @@ class pxContext {
                  bool downscaleSmooth = false,
                  pxConstantsMaskOperation::constants maskOp= pxConstantsMaskOperation::NORMAL);
 
-  void drawEffect(float x, float y, float w, float h, pxTextureRef t, shaderProgram *shader);
+  void drawEffect(float x, float y, float w, float h, pxTextureRef t, shaderProgram *shader, void *user = NULL);
+
+  struct filterXYR
+  {
+    float x; float y; float r;
+  };
+  
+  pxContextFramebufferRef applyBlurSettings(pxContextFramebufferRef src, shadowFx_t *shdw,
```

Comment:
Let's move filterXYR over to pxContextGL.cpp since the details are not needed here in the header.

Reply:
Done